### PR TITLE
update dependency versions

### PR DESCRIPTION
**Pull request by watashiSHUN**
make it work in VS with `.net.sdk.functions 1.0.13`
+reference https://github.com/Azure/azure-functions-vs-build-sdk/issues/189
+reference https://github.com/Azure/azure-functions-templates/issues/659

### CODE REVIEW

File: src/EventGridExtension/EventGridExtension.csproj [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
+    <TargetFramework>net46</TargetFramework>
```
Comment (paulbatum) [inline src/EventGridExtension/EventGridExtension.csproj]: hmm, why this change?

File: src/EventGridExtension/EventGridExtension.csproj [hunk 2]
```diff
@@ -18,14 +18,9 @@
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
   </PropertyGroup>
 
-  <ItemGroup Condition="$(TargetFramework)=='net46'">
```
Comment (paulbatum) [inline src/EventGridExtension/EventGridExtension.csproj]: wondering about this too

File: src/EventGridExtension/EventGridExtension.csproj (hunk 1 above)
Comment (watashiSHUN) [reply src/EventGridExtension/EventGridExtension.csproj]: since now I no longer have a one nuget package targeting both frameworks, I am following the conventions to separate version and dependencies:
image
image
